### PR TITLE
Acronym Vault + Script Launch Options + Fix Antora Build Script

### PR DESCRIPTION
**Pull request by budak7273**

### CODE REVIEW

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+=== Coffee Stain Studios (CSS)
+
+_Not_ https://www.w3.org/Style/CSS/Overview.en.html[cascading style sheets].
+
+https://www.coffeestainstudios.com/[Coffee Stain Studios]
+is the Swedish game developer that created Satisfactory.
+
+=== Satisfactory Mod Loader (SML)
+
+This is the community-created mod responsible for loading other mods and providing an easy way for them to register content, configurations, and more.
```
Comment (Feyko) [inline modules/ROOT/pages/CommunityResources/AcronymVault.adoc:24]: Does SML load mods anymore? I thought it didn't

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Although it is possible to create a mod that does not depend on SML, they are very uncommon, as there are few reasons to do so.
+
+Its public source code can be found on
+https://github.com/satisfactorymodding/SatisfactoryModLoader[GitHub].
+
+=== Satisfactory Mod Repository (SMR) aka Ficsit.app
+
+Ficsit.app, or the Satisfactory Mod Repository, is the website at which mods, this documentation, guides, and more are hosted.
```
Comment (Feyko) [inline modules/ROOT/pages/CommunityResources/AcronymVault.adoc:33]:
```suggestion
https://ficsit.app[ficsit.app], or the Satisfactory Mod Repository, is the website at which mods, this documentation, guides, and more are hosted.
```

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+=== Satisfactory Mod Loader (SML)
+
+This is the community-created mod responsible for loading other mods and providing an easy way for them to register content, configurations, and more.
+
+Although it is possible to create a mod that does not depend on SML, they are very uncommon, as there are few reasons to do so.
+
+Its public source code can be found on
+https://github.com/satisfactorymodding/SatisfactoryModLoader[GitHub].
+
+=== Satisfactory Mod Repository (SMR) aka Ficsit.app
```
Comment (Feyko) [inline modules/ROOT/pages/CommunityResources/AcronymVault.adoc:31]:
```suggestion
=== Satisfactory Mod Repository (SMR) aka ficsit.app
```

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+== In-Use Acronyms
+
+These acronyms are still actively in use.
+
+=== Coffee Stain Studios (CSS)
+
+_Not_ https://www.w3.org/Style/CSS/Overview.en.html[cascading style sheets].
+
+https://www.coffeestainstudios.com/[Coffee Stain Studios]
+is the Swedish game developer that created Satisfactory.
```
Comment (Feyko) [inline modules/ROOT/pages/CommunityResources/AcronymVault.adoc:20]:
```suggestion
is the Swedish game studio that created Satisfactory.
```

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Satisfactory Mod Manager handles all the steps of installing mods in Satisfactory for you!
+It works closely with SMR to function.
+SMM is only responsible for downloading the mods - it has no presence in a running game. SML is responsible for loading mod files at runtime.
+
+Its public source code can be found on
+https://github.com/satisfactorymodding/SatisfactoryModManager[GitHub].
+
+=== Satisfactory Save Editor (SSE)
+
+This name, and acronym, has been claimed by a number of save editor tools.
```
Comment (Feyko) [inline modules/ROOT/pages/CommunityResources/AcronymVault.adoc:51]:
```suggestion
This name, and acronym, has been claimed by a number of save editor tools. It is most often used when referring to Goz3rr's editor
```

File: modules/ROOT/pages/Development/BeginnersGuide/dependencies.adoc [hunk 6]
```diff
@@ -37,7 +37,7 @@ Continue through the rest of the prompts to install Visual Studio. Downloading a
 
 [WARNING]
 ====
-You will also need the .NET SDK, version 4.6 or higher (but not v5). If you do not know if you have that SDK, the VS installer should provide an option to install it. It is recommended to tick it.
+You will also need the .NET Framework, version 4.6 or higher (but not Core/v5). If you do not know if you have .Net Framework, the VS installer should provide an option to install it. It is recommended to tick it.
```
Comment (Feyko) [inline modules/ROOT/pages/Development/BeginnersGuide/dependencies.adoc:40]:
```suggestion
You will also need the .NET Framework SDK, version 4.6 or higher (but not Core/v5). If you do not know if you have the .NET Framework SDK, the VS installer should provide an option to install it. It is recommended to tick it.
```

**Review by Feyko — CHANGES_REQUESTED** [review]
Suggested patches (directly from GitHub, but I haven't touched formatting so it should be fine) and asked one question. Thanks for the PR!

File: modules/ROOT/pages/CommunityResources/AcronymVault.adoc (hunk 1 above)
Comment (budak7273) [reply modules/ROOT/pages/CommunityResources/AcronymVault.adoc]: Fixed in 1dcbcda

**Review by Feyko — APPROVED** [review]

**Pull request merged**